docs/data-product-studio/snowtype/snowtype-config/index.md

```diff
@@ -83,6 +83,23 @@ options?: {
 }
 ```
 
+### `namespace`
+
+:::info
+
+This option only applies when generating Swift
```

Comment:
`This option only applies when generating Swift` -> `This option only applies when generating Swift **code**` ->